go/systests/team_invite_test.go

```diff
@@ -479,14 +479,45 @@ func TestSweepObsoleteKeybaseInvites(t *testing.T) {
 	// TestObsoletingInvites*.
 	ann.addTeamMember(team, bob.username, keybase1.TeamRole_WRITER)
 
+	// Bob then leaves team.
+	bob.leave(team)
+
 	teamObj, err := teams.Load(context.Background(), ann.tc.G, keybase1.LoadTeamArg{
 		Name:        team,
 		ForceRepoll: true,
 	})
 	require.NoError(t, err)
-	_ = teamObj
 
-	hasInvite, err := teamObj.HasActiveInvite(bob.userVersion().TeamInviteName(), "keybase")
+	// Invite should be obsolete.
+	invite, err := teamObj.FindActiveInvite(bob.userVersion().TeamInviteName(), "keybase")
```

Comment:
I think a function called `FindActiveInvite` should not return an obsolete invite. This is otherwise prone to misuse. It should filter them out. There can be FindObsoleteInvite which is basically only for tests right?